CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # activerecord-postgres_pub_sub
 
+## v1.2.1 (unreleased)
+- Fix Appraisals version constraints for Rails 6.0.
```

Comment:
I don't think this requires a changelog entry or a version bump. This is a change to unreleased, internal, test code. 

Users of the gem shouldn't need to be aware of this.

Reply:
makes sense, will remember that for next time